{"uid":"3b4f5bc71094365f","name":"test_pitr_backup_aws","fullName":"tests.integration.test_backups_pitr#test_pitr_backup_aws","historyId":"ef43715c92cd9caf093556bb9ee9cad6","time":{"start":1728523905997,"stop":1728524972132,"duration":1066135},"description":"Build and deploy two units of PostgreSQL in AWS and then test PITR backup and restore actions.","descriptionHtml":"<p>Build and deploy two units of PostgreSQL in AWS and then test PITR backup and restore actions.</p>\n","status":"broken","statusMessage":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 97, in pytest_runtest_teardown\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 100, in pytest_runtest_teardown\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 853, in pytest_runtest_teardown\n    yield from self._runtest_for(item, \"teardown\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 885, in pytest_runtest_teardown\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 189, in pytest_runtest_teardown\n    item.session._setupstate.teardown_exact(nextitem)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 557, in teardown_exact\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 546, in teardown_exact\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1032, in finish\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1021, in finish\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/tests/integration/test_backups_pitr.py\", line 91, in cloud_configs\n    for bucket_object in bucket.objects.filter(Prefix=config[\"path\"].lstrip(\"/\")):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 79, in __iter__\n    for page in self.pages():\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 169, in pages\n    for page in pages:\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 269, in __iter__\n    response = self._make_request(current_kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 357, in _make_request\n    return self._method(**current_kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 569, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1005, in _make_api_call\n    http, parsed_response = self._make_request(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1029, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 119, in make_request\n    return self._send_request(request_dict, operation_model)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 196, in _send_request\n    request = self.create_request(request_dict, operation_model)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 132, in create_request\n    self._event_emitter.emit(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 412, in emit\n    return self._emitter.emit(aliased_event_name, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 256, in emit\n    return self._emit(event_name, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 239, in _emit\n    response = handler(**kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 105, in handler\n    return self.sign(operation_name, request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 197, in sign\n    auth.add_auth(request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/auth.py\", line 423, in add_auth\n    raise NoCredentialsError()\nbotocore.exceptions.NoCredentialsError: Unable to locate credentials","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1728523905496,"stop":1728523905497,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1728523905497,"stop":1728523905921,"duration":424},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"github_secrets","time":{"start":1728523905497,"stop":1728523905497,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728523905497,"stop":1728523905497,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"cloud_configs","time":{"start":1728523905922,"stop":1728523905996,"duration":74},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728523905997,"stop":1728523905997,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy two units of PostgreSQL in AWS and then test PITR backup and restore actions.","status":"broken","statusMessage":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 97, in pytest_runtest_teardown\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 100, in pytest_runtest_teardown\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 853, in pytest_runtest_teardown\n    yield from self._runtest_for(item, \"teardown\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 885, in pytest_runtest_teardown\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 189, in pytest_runtest_teardown\n    item.session._setupstate.teardown_exact(nextitem)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 557, in teardown_exact\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 546, in teardown_exact\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1032, in finish\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1021, in finish\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/tests/integration/test_backups_pitr.py\", line 91, in cloud_configs\n    for bucket_object in bucket.objects.filter(Prefix=config[\"path\"].lstrip(\"/\")):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 79, in __iter__\n    for page in self.pages():\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 169, in pages\n    for page in pages:\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 269, in __iter__\n    response = self._make_request(current_kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 357, in _make_request\n    return self._method(**current_kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 569, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1005, in _make_api_call\n    http, parsed_response = self._make_request(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1029, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 119, in make_request\n    return self._send_request(request_dict, operation_model)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 196, in _send_request\n    request = self.create_request(request_dict, operation_model)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 132, in create_request\n    self._event_emitter.emit(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 412, in emit\n    return self._emitter.emit(aliased_event_name, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 256, in emit\n    return self._emit(event_name, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 239, in _emit\n    response = handler(**kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 105, in handler\n    return self.sign(operation_name, request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 197, in sign\n    auth.add_auth(request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/auth.py\", line 423, in add_auth\n    raise NoCredentialsError()\nbotocore.exceptions.NoCredentialsError: Unable to locate credentials","steps":[],"attachments":[{"uid":"2b26ec32d87f0dac","name":"log","source":"2b26ec32d87f0dac.txt","type":"text/plain","size":10985}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1728524972133,"stop":1728524972133,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"cloud_configs::finalizer","time":{"start":1728524972133,"stop":1728524973225,"duration":1092},"status":"broken","statusMessage":"botocore.exceptions.NoCredentialsError: Unable to locate credentials\n","statusTrace":"  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/tests/integration/test_backups_pitr.py\", line 91, in cloud_configs\n    for bucket_object in bucket.objects.filter(Prefix=config[\"path\"].lstrip(\"/\")):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 79, in __iter__\n    for page in self.pages():\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 169, in pages\n    for page in pages:\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 269, in __iter__\n    response = self._make_request(current_kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 357, in _make_request\n    return self._method(**current_kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 569, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1005, in _make_api_call\n    http, parsed_response = self._make_request(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1029, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 119, in make_request\n    return self._send_request(request_dict, operation_model)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 196, in _send_request\n    request = self.create_request(request_dict, operation_model)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 132, in create_request\n    self._event_emitter.emit(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 412, in emit\n    return self._emitter.emit(aliased_event_name, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 256, in emit\n    return self._emit(event_name, kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 239, in _emit\n    response = handler(**kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 105, in handler\n    return self.sign(operation_name, request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 197, in sign\n    auth.add_auth(request)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/auth.py\", line 423, in add_auth\n    raise NoCredentialsError()\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1728524973226,"stop":1728524973678,"duration":452},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728524973679,"stop":1728524973679,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728524973679,"stop":1728524973679,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728524973679,"stop":1728524973679,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_backups_pitr"},{"name":"host","value":"runner"},{"name":"thread","value":"15099-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_backups_pitr"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"14ed0db98efe6da2","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":15,"skipped":0,"passed":0,"unknown":0,"total":15},"items":[{"uid":"3a27928499b5402","reportUrl":"../105//#testresult/3a27928499b5402","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1728437466890,"stop":1728438433216,"duration":966326}},{"uid":"5e5417e4e153c211","reportUrl":"../104//#testresult/5e5417e4e153c211","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1728351103875,"stop":1728352091770,"duration":987895}},{"uid":"7a92ca64a71bd0ff","reportUrl":"../103//#testresult/7a92ca64a71bd0ff","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1728264617201,"stop":1728265644190,"duration":1026989}},{"uid":"eae7c5db9b2d716a","reportUrl":"../102//#testresult/eae7c5db9b2d716a","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1728178245049,"stop":1728179254164,"duration":1009115}},{"uid":"768db1ca408f212c","reportUrl":"../101//#testresult/768db1ca408f212c","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1728091850421,"stop":1728092791864,"duration":941443}},{"uid":"f6ee314ae1f04666","reportUrl":"../100//#testresult/f6ee314ae1f04666","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1728005472789,"stop":1728006524261,"duration":1051472}},{"uid":"171d9fd8d4c161b0","reportUrl":"../99//#testresult/171d9fd8d4c161b0","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727919067751,"stop":1727920069491,"duration":1001740}},{"uid":"8723a5b983c55883","reportUrl":"../98//#testresult/8723a5b983c55883","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727832674013,"stop":1727833642507,"duration":968494}},{"uid":"89ea34f586b3e79e","reportUrl":"../97//#testresult/89ea34f586b3e79e","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727746287248,"stop":1727747336686,"duration":1049438}},{"uid":"50717b3790f8d917","reportUrl":"../96//#testresult/50717b3790f8d917","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727659843315,"stop":1727660841456,"duration":998141}},{"uid":"5db0f380b90977c9","reportUrl":"../95//#testresult/5db0f380b90977c9","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727573460491,"stop":1727574373636,"duration":913145}},{"uid":"91f65f8d03df8e35","reportUrl":"../94//#testresult/91f65f8d03df8e35","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727487105421,"stop":1727488167335,"duration":1061914}},{"uid":"d957ad16132f5751","reportUrl":"../93//#testresult/d957ad16132f5751","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727400736539,"stop":1727401734585,"duration":998046}},{"uid":"b9e425f2f5b76ad1","reportUrl":"../92//#testresult/b9e425f2f5b76ad1","status":"broken","statusDetails":"botocore.exceptions.NoCredentialsError: Unable to locate credentials","time":{"start":1727314379378,"stop":1727315375377,"duration":995999}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"3b4f5bc71094365f.json","parameterValues":[]}